### PR TITLE
Russian translation

### PR DESCRIPTION
**Pull request by Ujjwal-soni98**
Russian translation. Related to #10.

### CODE REVIEW

File: SignalAnalysis/localization/strings.ru-RU.resx [hunk 1]
```diff
@@ -0,0 +1,522 @@
...
+    <value>Запомнить предыдущий путь диалога открытия/сохранения</value>
+  </data>
+  <data name="strChkEntropy" xml:space="preserve">
+    <value>Энтропия (приблизительно &amp;&amp; выборка)</value>
+  </data>
+  <data name="strChkPower" xml:space="preserve">
+    <value>Мощность (дБ)</value>
+  </data>
+  <data name="strDlgReset" xml:space="preserve">
+    <value>Вы хотите сбросить все поля
```
Comment (arthurits) [inline SignalAnalysis/localization/strings.ru-RU.resx:154]: Could you please verify that the line break is correctly placed?

File: SignalAnalysis/localization/strings.ru-RU.resx [hunk 2]
```diff
@@ -0,0 +1,522 @@
...
+  <data name="strMsgBoxErrorSaveData" xml:space="preserve">
+    <value>При сохранении файла данных произошла непредвиденная ошибка.
+Повторите попытку позже или обратитесь к инженеру-программисту.
+{0}</value>
+  </data>
+  <data name="strMsgBoxErrorSaveDataTitle" xml:space="preserve">
+    <value>Ошибка сохранения данных</value>
+  </data>
+  <data name="strMsgBoxExit" xml:space="preserve">
+    <value>Вы уверены, что хотите выйти
```
Comment (arthurits) [inline SignalAnalysis/localization/strings.ru-RU.resx:317]: Could you please verify that the line break is correctly placed?

File: SignalAnalysis/localization/strings.ru-RU.resx [hunk 3]
```diff
@@ -0,0 +1,522 @@
...
+  </data>
+  <data name="strMsgBoxOKSaveData" xml:space="preserve">
+    <value>Данные успешно сохранены на диск.</value>
+  </data>
+  <data name="strMsgBoxOKSaveDataTitle" xml:space="preserve">
+    <value>Данные сохранены</value>
+  </data>
+  <data name="strMsgBoxTaskCancel" xml:space="preserve">
+    <value>Прекращен расчет фрактальной размерности
+Хаусдорфа-Безиковича.</value>
```
Comment (arthurits) [inline SignalAnalysis/localization/strings.ru-RU.resx:344]: Could you please verify that the line break is correctly placed?

**Review by arthurits — CHANGES_REQUESTED** [review]
Please have a look at the 3 comments regarding the correct placement of line breaks.

File: SignalAnalysis/localization/strings.ru-RU.resx (hunk 3 above)
Comment (Ujjwal-soni98) [reply SignalAnalysis/localization/strings.ru-RU.resx:344]: @arthurits what type of line breaks?can u elaborate. i'm not getting it.

Comment (arthurits) [reply SignalAnalysis/localization/strings.ru-RU.resx:344]: What I mean is that, for instance, I broke the string "Прекращен расчет фрактальной размерности Хаусдорфа-Безиковича." in two lines (as that text will be part of a MessageBox) as you can see in the code above (lines 343 and 344). 
I'm not sure that the correct place to do the breaking.
There are other line breaks I'm not sure about: lines 154-155 and 317-318.

**Pull request merged**